programs/github

```diff
@@ -2,7 +2,7 @@ local github = dofile("apis/github")
 
 local action = select(1, ...)
 
-local usage = "Usage: github clone <user>/<repo> <destination>"
+local usage = "Usage: github clone <user>/<repo> <branch||tag> <destination>"
```

Comment:
Going for docopt style here, so `[<branch>|<tag>]`. Or does this work with any "commitish" object, like `f572e949b87a8050ff159c82b16315a24665af83`?